schemas/oic.linkstatus-rw-schema.json

```diff
@@ -0,0 +1,46 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description" : "Copyright (c) 2018 Open Connectivity Foundation, Inc. All rights reserved.",
+    "id": "http://openconnectivityfoundation.github.io/core/schemas/oic.linkstatus-rw-schema.json#",
+    "title": "Linkstatus Resource Type",
+    "definitions": {
+      "linkstatus-properties": {
+        "properties": {
+          "linkstate": {
+            "type": "string",
+            "enum": [ "init", "active", "error", "timeout" ],
+            "readOnly": true,
+            "description": "The state of the dynlink"
+          },
+          "laststatus": {
+            "type": "number",
+            "enum": [200, 201, 400, 403, 404, 405, 406, 408, 409, 410, 415, 418, 429, 500, 501, 502, 503, 504],
```

Comment:
There are more values than just these, right?  Do we really want to enumerate all possible values?  We have other strings that convey the last error code like for the OCF Cloud Resources and they are not enumerations but just a "string".